helm-chart/templates/nextcloud-aio-apache-persistentvolumeclaim.yaml

```diff
@@ -9,7 +9,7 @@ spec:
   storageClassName: {{ .Values.STORAGE_CLASS }}
   {{- end }}
   accessModes:
-    - ReadWriteMany
+    - ReadWriteOnce
   resources:
     requests:
-      storage: {{ .Values.MAX_STORAGE_SIZE }}
+      storage: 100Mi
```

Comment:
Looks like this wasn't replaced anymore

Reply:
i initially didnt intend to make all volumes configurable but found a solution with https://github.com/nextcloud/all-in-one/pull/1848/commits/b6c3a777b0628cf3116b045bbade24e440d04eb9

Reply:
Ugh that solution is not good. Please just make every volume configurable. This is how helm charts work, they are complex and configurable :/

Reply:
Okay, I will add one for each pvc

Reply:
fixed with https://github.com/nextcloud/all-in-one/pull/1848/commits/133b149ee3272d6ac7815b3230edc5fd7215617a

Reply:
improved with https://github.com/nextcloud/all-in-one/pull/1848/commits/133b149ee3272d6ac7815b3230edc5fd7215617a